src/index.html

```diff
@@ -11,8 +11,63 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@700&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="container"
+      data-qa="card"
+    >
+      <div class="container__imac">
+        <img
+          class="container__imac-image"
+          src="./images/imac.jpeg"
+          alt="imac-image"
+        />
+      </div>
+
+      <div class="container__link">
+        <a
+          href="#"
+          class="container__link-text"
+        >
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
+      </div>
+
+      <div class="container__code">Product code: 195434</div>
+
+      <div class="container__stars">
+        <div class="container__stars-star"></div>
+        <div class="container__stars-star"></div>
+        <div class="container__stars-star"></div>
+        <div class="container__stars-star"></div>
+        <div class="container__stars-star"></div>
+        <div class="container__stars-text">Reviews: 5</div>
+      </div>
```

Comment:
This text do not belong stars block
```suggestion
      <div class="container__ratings">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
        <p class="container__reviews">Reviews: 5</p>
      </div>
```